appcues/src/main/java/com/appcues/trait/appcues/BackgroundContentTrait.kt

```diff
@@ -36,12 +37,12 @@ internal class BackgroundContentTrait(
     private val content = config.getConfigPrimitive("content")
 
     @Composable
-    override fun BoxScope.DecorateStep(stepDecoratingPadding: StepDecoratingPadding) {
+    override fun BoxScope.DecorateStep(wrapperInsets: PaddingValues, stickyContentPadding: StickyContentPadding) {
```

Comment:
the trait dont necessarily needs to apply the safe area, its just there to be used if necessary